doc/devdocs/modules/launcher/plugins/indexer.md

```diff
@@ -36,4 +36,4 @@ Each of the indexer plugin results has a score set to 0 so they are present at t
 [OLEDBCommand]: https://docs.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbcommand?view=dotnet-plat-ext-3.1
 [OLEDBConnection]: https://docs.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbconnection?view=dotnet-plat-ext-3.1
 [Contains]: https://docs.microsoft.com/en-us/windows/win32/search/-search-sql-contains
-[Like]: https://docs.microsoft.com/en-us/windows/win32/search/-search-sql-like
\ No newline at end of file
+[Like]: https://docs.microsoft.com/en-us/windows/win32/search/-search-sql-like
```

Comment:
What changed here

Reply:
Oh, I delete space in the end of the line.